src/components/UI/FormControl/index.css

```diff
@@ -93,6 +93,10 @@
   border-bottom-right-radius: 0;
 }
 
+.FormControl--segmented > button > svg {
+  display: block;
+}
```

Comment:
the svg icons where not properly sentered (svg images are inline by default and it was adding a few empty pixels under it)